app/views/categories/_equipment_models_table.html.erb

```diff
@@ -0,0 +1,40 @@
+<table id="table_woo" class="datatable table table-striped table-bordered">
+  <thead>
+  <tr>
+    <th>Name</th>
+    <th class="no_sort">Current Status</th>
+    <th>Available</th>
+    <th>Reserved</th>
+    <th>Checked-Out</th>
+    <th>Overdue</th>
+    <th>Total Stock</th>
+  <% if can? :manage, EquipmentModel %>
+    <th class="no_sort"> </th>
+    <th class="no_sort"> </th>
+  <% end %>
+  </tr>
+  </thead>
+  <tbody>
+  <% for equipment_model in @category.equipment_models %>
+    <tr>
+      <td><%= link_to equipment_model.name, equipment_model %></td>
+      <% if equipment_model.availability(Date.today) == 'all' %>
```

Comment:
We need to use `Time.zone.today` for all of these to ensure we're properly accounting for the time zone.